wallet/src/main/java/com/android/identity_credential/wallet/ui/destination/document/DocumentInfoScreen.kt

```diff
@@ -342,7 +352,7 @@ fun DocumentInfoScreen(
                 Image(
                     bitmap = documentInfo.issuerLogo.asImageBitmap(),
                     contentDescription = stringResource(R.string.accessibility_artwork_for, documentInfo.issuerName),
-                    modifier = Modifier.height(150.dp).fillMaxSize()
+                    modifier = Modifier.height(150.dp)
```

Comment:
What is this about?

Reply:
fillMaxWidth() already handles horizontal space: The Row has Modifier.fillMaxWidth(), so it will already take up the entire width available to it.

Reply:
I have put it back but it has no effect.